packages/trader/src/RangeTrader.ts

```diff
@@ -125,6 +125,10 @@ export class RangeTrader {
       desiredPrice: this.formatDecimalString(this.normalizePriceFeedDecimals(desiredPrice))
     });
 
+    if (this.tokenPriceFeed.invertPrice === "true") {
```

Comment:
why not just `if (this.tokenPriceFeed.invertPrice)`

Reply:
it's a string not a bool!